SPECS/python-async-generator/python-async-generator.spec

```diff
@@ -0,0 +1,103 @@
+# what it's called on pypi
+%global srcname async_generator
+# what it's imported as
+%global libname async_generator
+# name of egg info directory
+%global eggname async_generator
+# package name fragment
+%global pkgname async-generator
+
+%global _description \
+This library generally tries hard to match the semantics of Python 3.6's native\
+async generators in every detail (PEP 525), with additional support for yield\
+from and for returning non-None values from an async generator (under the\
+theory that these may well be added to native async generators one day).
+
+Summary:        Async generators and context managers
+Name:           python-%{pkgname}
+Version:        1.10
+Release:        10%{?dist}
+License:        MIT or ASL 2.0
+URL:            https://github.com/python-trio/async_generator
+Vendor:         Microsoft
```

Comment:
>Microsoft [](http://example.com/codeflow?start=16&length=9)

Nit: I believe we've standardized on "Microsoft Corporation"

Reply:
fixed:)